tests/external/scar/test_scar.py

```diff
@@ -7,8 +7,11 @@
 def test_scar():
     n_latent = 5
     adata = synthetic_iid()
+
+    adata.obs["batch"] = adata.obs["batch"].cat.rename_categories(["batch_2", "batch_3"])
+
     adata.X = scipy.sparse.csr_matrix(adata.X)
-    SCAR.setup_anndata(adata)
+    SCAR.setup_anndata(adata, batch_key="batch")
```

Comment:
Please add label_key="batch". It's good to verify that handling label_key is not broken in the future.

Reply:
Looks like some other test cases the renaming is not strictly necessary. Should we keep that?